src/Administration/Resources/app/administration/src/app/service/custom-field.service.js

```diff
@@ -140,7 +140,7 @@ export default function createCustomFieldService() {
     }
 
     function upsertType(name, configuration) {
-        $typeStore[name] = { ...$typeStore[name], ...{ configuration } };
+        $typeStore[name] = { ...$typeStore[name], ...configuration };
```

Comment:
Can you add a small test case for this? As the service has no other dependencies it should possible to write a unit test that verifies that after `upserting` the values are correctly stored in the store

Reply:
Sure, I can write a corresponding test. For your information, so far this service has not been tested.

@keulinho Just out of interest: Why am I the only one who has to write tests? In other pull requests these are not requested and approved.

Reply:
Thx :+1: 

Others also have to write tests when they touch admin js parts ;) especially as you were touching a service class and not "only" a single component